parfive/downloader.py

```diff
@@ -321,7 +321,7 @@ def callback(token, future, main_pb):
 
     @staticmethod
     async def _get_http(session, *, url, filepath_partial, chunksize=100,
-                        file_pb=None, token, overwrite, timeouts, **kwargs):
+                        file_pb=None, token, overwrite, timeouts, max_splits=12, **kwargs):
```

Comment:
are we planning on exposing this to users?

Reply:
Yup, advanced users should be able to use it. Not regular users.

Reply:
So where are we going to put the option? This is basically completely inaccessible right?

Reply:
Nope, `enqueue_file`'s kwargs gets passed down to this. This is meant for advanced users who really want to do this.

Reply:
Ok that's great.